### DISCUSSION TITLE
Does the tour library support auto skiping a stop if the target is no available

### DISCUSSION THREAD

**Discussion by jiyang-sas** [gilbarbara/react-joyride#633] [Q&A] [answered]
Does the tour library support the following:
1) Auto skiping a stop if the target is no available/visible?
2) Intro / Outro dialogs (non-targetted)
3) Ability to perform an action either on entrance to a stop or exit from a stop

Thanks a lot!

**Comment by gilbarbara** [comment, marked as answer]
hey @jiyang-sas 

Did you read the documentation?
By stop, you mean **step**?

1. yes
2. with the "center" placement
3. using the callback

**Reply by jiyang-sas** [reply]
Thanks a lot!
